assets/pages/new-contributions-landing/components/SetPasswordForm.jsx

```diff
@@ -131,19 +132,20 @@ function SetPasswordForm(props: PropTypes) {
           errorMessage="Please enter a password between 6 and 20 characters long"
           required
         />
-        <ButtonWithRightArrow
-          componentClassName={classNameWithModifiers('form__submit', ['create-account'])}
-          buttonClassName={classNameWithModifiers('form__submit-button', ['create-account'])}
-          accessibilityHintId="accessibility-hint-create-account"
-          type="submit"
-          buttonCopy="Create an account"
-        />
-        <ButtonWithRightArrow
-          componentClassName={classNameWithModifiers('form__submit', ['no-thanks'])}
-          buttonClassName={classNameWithModifiers('form__submit-button', ['no-thanks'])}
-          accessibilityHintId="accessibility-hint-no-thanks"
-          type="button"
-          buttonCopy="No, thank you"
+        <div className="form__submit form__submit--set-password">
```

Comment:
No longer using a component to make this consistent with landing page. Next step will be to make both use a core component shared across site.

Reply:
I don't think we should move away from using the `ButtonWithRightArrow` class, I don't see the benefit. This is only consistent with the landing page anyway in the sense that we're not using a component, but I don't think that's worth the added noise of writing the markup here. 

Also, the buttons are so similar, I think it's confusing that they have different implementations (one just pure markup, one using `CtaLink`

Reply:
I would just keep `ButtonWithRightArrow` for this PR, then make the landing page use it (or a modified version where the arrow is optional) in the future

Reply:
But I'd be happy with going back to using the component, for now, if you'd prefer